dohproxy/utils.py

```diff
@@ -243,6 +243,12 @@ def proxy_parser_base(*, port: int,
         action='version',
         version='%(prog)s {}'.format(__version__),
     )
+    parser.add_argument(
```

Comment:
this option should only belong to the httpproxy code. Can you move this within httpproxy.py:parse_args pls